test/test_qspdlog.cpp

```diff
@@ -145,6 +145,20 @@ private slots:
         QCOMPARE(widget.itemsCount(), 20);
     }
 
+    void backgroundForegroundColorTest()
+    {
+        QSpdLog widget;
+        QCOMPARE(widget.getBackgroundMapping("test"), std::nullopt);
+        widget.setBackgroundMapping("test", QBrush(Qt::red));
+        QCOMPARE(widget.getBackgroundMapping("test"), QBrush(Qt::red));
+        QCOMPARE(widget.getBackgroundMapping("test2"), std::nullopt);
```

Comment:
You are setting the value for the `test2` logger, that's why the `erase` case is not covered.

Reply:
Thanks